ui/oopsyraidsy/data/07-dt/dungeon/meso-terminal.ts

```diff
@@ -0,0 +1,79 @@
+import NetRegexes from '../../../../../resources/netregexes';
+import ZoneId from '../../../../../resources/zone_id';
+import { OopsyData } from '../../../../../types/data';
+import { OopsyTriggerSet } from '../../../../../types/oopsy';
+import { playerDamageFields } from '../../../oopsy_common';
+
+export type Data = OopsyData;
+
+const triggerSet: OopsyTriggerSet<Data> = {
+  zoneId: ZoneId.TheMesoTerminal,
+  damageWarn: {
+    // ** Adds Pre-Boss 1 ** //
+    'Meso Terminal Terminal Watchman Electric Shock': 'AC08', // Targeted circle AoE
+    'Meso Terminal Preserved Medic Alexandrian Gravity': 'AC09', // Targeted large circle AoE
+
+    // ** Chirurgeon General ** //
+    'Meso Terminal Chirurgeon General Biochemical Front': 'AB1A', // Frontal half-room cleave
+    'Meso Terminal Chirurgeon General Sterile Sphere Large': 'AB1D', // Large circle AoEs with timer
+    'Meso Terminal Chirurgeon General Sterile Sphere Small': 'AB1C', // Small circle AoEs with timer
+
+    // ** Adds Pre-Boss 2 ** //
+    'Meso Terminal Terminal Claw Pressure Wave': 'AC0C', // Frontal cone AoE
+    'Meso Terminal Preserved Prisoner Rusted Knives': 'AC0D', // Chariot AoE
+    'Meso Terminal Preserved Prisoner Overpower': 'AC0E', // Chariot AoE
+
+    // ** Executioners ** //
+    'Meso Terminal Headsman Shackles Of Fate Error': 'AA3F', // Leaving the chain circle
+    'Meso Terminal Headsman Dismemberment': 'AA43', // Line AoE
+    'Meso Terminal Headsman Flaying Flail': 'AA48', // Triple circle AoE
+    'Meso Terminal Headsman Peal Of Judgment': 'AA4A', // Pathing lightning AoEs
+    'Meso Terminal Headsman Chopping Block': 'AA4B', // Chariot AoE
+    'Meso Terminal Headsman Execution Wheel': 'AA4C', // Dynamo AoE
+    'Meso Terminal Pale Headsman Will Breaker': 'AF38', // Interruptible tank vuln
```

Comment:
There should probably be something here for the cleansable healer Doom, if possible.